htcrystalball/__init__.py

```diff
@@ -10,11 +10,11 @@
 logging.basicConfig(level=logging.ERROR)
 
 # Internal logging level
-logger = logging.getLogger('crystal_balls')
-logger.setLevel(level=logging.DEBUG)
+LOGGER = logging.getLogger('crystal_balls')
+LOGGER.setLevel(level=logging.DEBUG)
```

Comment:
Here I made a mistake in my PR. It is best to remove the logger from `__init__.py` and instantiate it in `main.py` instead. 

The line `logger = logging.getLogger('crystal_balls')` will get the logger and doesn't require `LOGGER` to be a global. Read [this post](https://stackoverflow.com/questions/7621897/python-logging-module-globally) for how it *should* be done.

Reply:
Sounds good. But that change should be done in its own PR, rather than expanding the scope of this one.

Reply:
I agree. Would it in this case be advisable to undo the changes to make `logger` a global?

Reply:
It would be cleaner if it didn't ping-pong around, but I don't want it to be a blocker.